src/pint/models/astrometry.py

```diff
@@ -903,25 +904,73 @@ def as_ECL(self, epoch=None, ecl="IERS2010"):
         epoch : `astropy.time.Time` or Float, optional
             new epoch for position.  If Float, MJD(TDB) is assumed.
             Note that uncertainties are not adjusted.
+        ecl : str, optional
+            Obliquity for PulsarEcliptic frame
 
         Returns
         -------
         pint.models.astrometry.AstrometryEcliptic
-
-        Raises
-        ------
-        ValueError
-            If obliquity is already set and a different obliquity is requested
         """
-        if not (ecl == self.ECL.value):
-            raise ValueError(
-                "Cannot convert from obliquity '%s' to requested obliquity '%s'"
-                % (self.ECL.value, ecl)
-            )
-        m = copy.deepcopy(self)
-        if epoch is not None:
-            m.change_posepoch(epoch)
-        return m
+
+        # change epoch only
+        if ecl == self.ECL.value:
+            print("here")
```

Comment:
Is this a left over debug line?

Reply:
Yes, that was foolish.  I'll fix and push that back